assets/javascript/app.js

```diff
@@ -149,6 +149,14 @@ var movieArray = [
   { name: 'Wives Under Suspicion', year: 1938 }
 ];
 
+movieArray.sort(function (a,b){
```

Comment:
What does this do?

Reply:
Nothing for now, I forgot to comment it out.